src/org/zaproxy/zap/utils/Stats.java

```diff
@@ -26,128 +26,179 @@
 import java.util.Map;
 import java.util.Map.Entry;
 
+import org.apache.log4j.Logger;
+
 public class Stats {
 
-	private static Statistics stats = new Statistics();
-	private static Map<String, Statistics> siteStats = new HashMap<String, Statistics>();
```

Comment:
There are a couple of imports that could be removed (Collections, HashMap, Map, Map.Entry).